
# region Motors are connected to the following GPIO pins
# Left Motors
GPIO13_PWM1 = 33
GPIO19_PWM1 = 35
GPIO2 = 3
GPIO3 = 5
GPIO4 = 7
GPIO14 = 8
GPIO15 = 10

# Right Motors
GPIO12_PWM0 = 33
GPIO18_PWM0 = 35
GPIO17 = 11
GPIO27 = 13
GPIO22 = 15
GPIO23 = 16
GPIO24 = 18
# endregion

# region IR array is connected to the following GPIO pins
GPIO10 = 19
GPIO21 = 21
GPIO11 = 23
# endregion

##### Misc values #####
MAX_SPEED_MOTOR = 100
# TODO: is 50 enough?
PWM_FREQUENCY = 50

DIR_FORWARD = 1
DIR_BACKWARD = -1
